### ISSUE TITLE
New SDXS model has a UNETLoader error?

### ISSUE THREAD

**Issue by Wraithnaut** [comfyanonymous/ComfyUI#3249]
A new SDXS model was released but I haven't gotten it to work yet with the basic 0.9 workflow I had.

> I upload a new version of SDXS-512 for community. I am glad if you can try it.

> https://huggingface.co/IDKiro/sdxs-512-dreamshaper

> _Originally posted by @IDKiro in https://github.com/comfyanonymous/ComfyUI/issues/3167#issuecomment-2049634327_
            

Error:

```
Error occurred when executing UNETLoader:

ERROR: Could not detect model type of: /mnt/sdb3/ComfyUI-2024-04/models/unet/sdxs-512-dreamshaper_unet.safetensors

File "/mnt/sdb3/ComfyUI-2024-04/execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/mnt/sdb3/ComfyUI-2024-04/execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/mnt/sdb3/ComfyUI-2024-04/execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/mnt/sdb3/ComfyUI-2024-04/nodes.py", line 814, in load_unet
model = comfy.sd.load_unet(unet_path)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/mnt/sdb3/ComfyUI-2024-04/comfy/sd.py", line 600, in load_unet
raise RuntimeError("ERROR: Could not detect model type of: {}".format(unet_path))
```

**Comment by KaruroChori** [comment]
This was fixed as discussed in https://github.com/comfyanonymous/ComfyUI/issues/3147#issuecomment-2053559428

**Comment by Wraithnaut** [comment]
> This was fixed as discussed in [#3147 (comment)](https://github.com/comfyanonymous/ComfyUI/issues/3147#issuecomment-2053559428)

Awesome, marking as closed.
